sdk/sqlvirtualmachine/arm-sqlvirtualmachine/src/models/index.ts

```diff
@@ -93,6 +93,55 @@ export interface Resource {
   readonly type?: string;
 }
 
+/** Common error response for all Azure Resource Manager APIs to return error details for failed operations. (This also follows the OData error response format.). */
+export interface ErrorResponse {
```

Comment:
Pls check if we have other same definition

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/21422